lib/rake/task_manager.rb

```diff
@@ -301,6 +321,33 @@ def get_description(task)
       desc
     end
 
+    # Return a value representing the "cost" of transforming str1 into str2
+    def levenshtein_distance(str1, str2)
```

Comment:
If we merge this now I think we need to consider using `Gem::Text` instead of copy&paste